packages/cw20/Cargo.toml

```diff
@@ -17,3 +17,5 @@ serde = { version = "1.0.103", default-features = false, features = ["derive"] }
 
 [dev-dependencies]
 cosmwasm-schema = { version = "1.0.0" }
+cw20-base = { version = "0.13.4", path = "../../contracts/cw20-base" }
+cw-multi-test = { version = "0.13.4", path = "../multi-test" }
```

Comment:
`cw20-base` creates a circular dependency.
Both of those are not used anywhere.
```suggestion
```

Reply:
Ah - good catch. Thanks. Leftover from a scratched testing approach.